stacks/spica/src/bucket/pages/index/index.component.html

```diff
@@ -174,7 +174,7 @@ <h5>
             mat-resize-header
             [persist-header-width]="schema._id"
           >
-            id
+            _id
```

Comment:
```suggestion
            id
```

The end-user does not need to know that this field has leading underscore. Can we revert this change?

Reply:
User can create a field  named as **id** but not **_id**. If user creates **id** field there will be 2 id field and it will be more confusing. Also guide responses include **_id** instead of **id**. I think these changes will make things more understandable for users.

Reply:
Yes that makes sense